--- 1	2025-05-16 10:23:14.171419246 +0300
+++ 2	2025-05-16 10:23:07.056195103 +0300
@@ -2016,6 +2016,8 @@ u64 snd_usb_interface_dsd_format_quirks(
 
 	case USB_ID(0x0d8c, 0x0316): /* Hegel HD12 DSD */
 	case USB_ID(0x10cb, 0x0103): /* The Bit Opus #3; with fp->dsd_raw */
+	case USB_ID(0x10cb, 0x080b): /* The Bit Opus #11; with fp->dsd_raw */
+        case USB_ID(0x262a, 0x9023): /* Savitech SA9023; with fp->dsd_raw */
 	case USB_ID(0x16d0, 0x06b2): /* NuPrime DAC-10 */
 	case USB_ID(0x16d0, 0x06b4): /* NuPrime Audio HD-AVP/AVA */
 	case USB_ID(0x16d0, 0x0733): /* Furutech ADL Stratos */
